test/api/reports_controller_test.rb

```diff
@@ -11,7 +11,35 @@ class ReportsControllerTest < ActionDispatch::IntegrationTest
 
   before(:all) do
     @user = FactoryBot.create(:api_user, test_array: @@users_to_clean)
```

Comment:
Out of curiosity, is there a way to share `before(:all)` between files?  At a quick glance, this `@user` line looks the same across all tests here; perhaps other lines are shared too.

Reply:
No, not that I know of.  It has to do with order of precedence for includes/requires.  The only way I would know how do it would be to add a helper and use `require` at the top of the test, but then the `include MiniTest::Hooks` declaration would be missing and the whole thing will error out.  I've tried similar things before and it always barfs.